ads/_config.js

```diff
@@ -189,6 +190,7 @@ export const adPreconnect = {
     'https://prg.kargo.com',
   ],
   pulsepoint: 'https://tag.contextweb.com',
+  amoad: 'https://j.amoad.com',
```

Comment:
No, actually what I meant was that `https://j.amoad.com` would not need to be here because it is already mentioned above in the preload and we always preconnect to those hosts.